docs/ref/models/fields.txt

```diff
@@ -325,6 +325,31 @@ characters that aren't allowed in Python variable names -- notably, the
 hyphen -- that's OK. Django quotes column and table names behind the
 scenes.
 
+
+``db_comment``
+--------------
+
+.. versionadded:: 4.2
+.. attribute:: Field.db_comment
+
+A string describing the Model field which will be saved as an SQL
+COMMENT on the SQL COLUMN
+
+Unlike ``help_text``, db_comments are displayed in the database as
+SQL comments, which is useful for documenting fields
+for database administrators, developers, analysts, and others with
+direct database access, especially if you need to provide different
+explanations for users or site admins (using ``help_text``) compared to DBAs
+(using ``db_comment``). For example::
+
+    db_comment = "User karma points. This is field is denormalized,\
+                          so should not be updated alone. This comment is\
+                          managed by Django, and will be updated on every\
+                          database migration."
+
+``db_comment`` has a character limit that differs for each database
+backend. PostgreSQL has a 1GB limit. MySQL has a 1024 character limit.
```

Comment:
I wonder if this part. Ould be generalised?

As written I'm asking what about oracle / sqlite? Do these limits change between versions too? That could be all too much for the docs.